packages/babel-cli/src/babel/dir.js

```diff
@@ -1,5 +1,6 @@
 import defaults from "lodash/defaults";
 import outputFileSync from "output-file-sync";
+import { sync as mkdirpSync } from "mkdirp";
```

Comment:
Looks like this needs to be added to the `package.json`.

Reply:
Sure I'll do this.